src/js/components/Block/Container.tsx

```diff
@@ -1,63 +1,42 @@
 
-const ERROR_MESSAGE = <Alert ml={4} status='error'><AlertIcon /><AlertTitle>An error occurred while rendering this block.</AlertTitle></Alert>;
+const ErrorMessage = ({ error }) => {
+  return (<Alert ml={4} status='error'>
+    <AlertIcon />
+    <AlertTitle>An error occurred while rendering this block.</AlertTitle>
+    <AlertDescription>{error}</AlertDescription>
+  </Alert>)
+};
 
-// Don't open the context menu on these elements
-const CONTAINER_CONTEXT_MENU_FILTERED_TAGS = ["A", "BUTTON", "INPUT", "TEXTAREA", "LABEL", "VIDEO", "EMBED", "IFRAME", "IMG"];
+export const Container = forwardRef((props, ref) => {
+  const { children, isActive, isEditing, isHoveredNotChild, isDragging, isSelected, isOpen, hasChildren, uid, childrenUids, ...rest } = props;
 
-const isEventTargetIsCurrentBlockNotChild = (target: HTMLElement, thisBlockUid: string): boolean => {
-  if (!target) return false;
-
-  // if hovered element's closest block container has the current UID,
-  // we're hovering the current block. Otherwise return false
-  const closestBlockContainer = target.closest('.block-container') as HTMLElement;
-  return (closestBlockContainer?.dataset?.uid === thisBlockUid)
-}
-
-const _Container = React.forwardRef(({ children, isDragging, isSelected, isOpen, hasChildren, hasPresence, isLinkedRef, uid, childrenUids, menu, actions, reactions, isEditing, ...props }, ref) => {
-  const [isHoveredNotChild, setIsHoveredNotChild] = React.useState(false);
-
-  const internalRef = React.useRef(null)
-  const refs = useMergeRefs(internalRef, ref)
-
-  const handleMouseOver = (e) => setIsHoveredNotChild(isEventTargetIsCurrentBlockNotChild(e.target, uid));
-  const handleMouseLeave = () => isHoveredNotChild && setIsHoveredNotChild(false);
-  const { addToContextMenu, getIsMenuOpen } = React.useContext(ContextMenuContext);
-  const isMenuOpen = getIsMenuOpen(internalRef);
-
-  const MenuItems = () => {
-    return menu
-  }
-
-  return <>
-    <Box
-      ref={refs}
+  return <ErrorBoundary FallbackComponent={ErrorMessage}>
+    <chakra.div
+      ref={ref}
       className={[
         "block-container",
         isDragging ? "is-dragging" : "",
         isSelected ? "is-selected" : "",
         isOpen ? "is-open" : "",
-        isMenuOpen && 'isMenuOpen',
+        isActive && 'is-active',
+        isEditing && 'is-editing',
         (hasChildren && isOpen) ? "show-tree-indicator" : "",
-        isLinkedRef ? "is-linked-ref" : "",
         isHoveredNotChild && "is-hovered-not-child",
-        hasPresence ? "is-presence" : "",
       ].filter(Boolean).join(' ')}
-      lineHeight="2em"
-      position="relative"
-      borderRadius="0.125rem"
-      bg={isMenuOpen ? "background.upper" : undefined}
-      justifyContent="flex-start"
-      flexDirection="column"
-      display="block"
-      background="var(--block-surface-color)"
       opacity={isDragging ? 0.5 : 1}
       data-uid={uid}
       data-childrenuids={childrenUids}
-      sx={{
+      __css={{
```

Comment:
`__css` is like `sx` but lower styling priority, so we can more easily override styles when needed. Works when we're building simple/raw components like divs, and not extending defaults like Button.